### ISSUE TITLE
How do I make inference?

### ISSUE THREAD

**Issue by sergiocanar** [OSUPCVLab/SegFormer3D#14]
After training the model, I'm encountering difficulties when attempting to test the model and visualize the predictions. Specifically, after running inference on the data, Im not able to match the results you guys showed in your repository.

**Comment by sergiocanar** [comment]
Sure! Im using Brats2017 that you guys uploaded in the repository. And after I pass one of the modalities.pt through the model I get the logits, but Im unsure to how obtain the final segmentation volume to visualize it.

**Comment by bnavard** [comment]
Which code are you using ? Can you put it here ?

**Comment by sergiocanar** [comment]
Sure! Its in my forked repo but this is how I extract the volumes from the logits 
[inference.txt](https://github.com/user-attachments/files/17681288/inference.txt)

**Comment by microchila** [comment]
@sergiocanar  hello. Do you have the full test script?

**Comment by lj-xh** [comment]
Hello, I am currently working on the medical image segmentation field and have completed the model training process. Could you please provide a corresponding testing script? Thank you very much!

**Comment by sergiocanar** [comment]
@microchila @lj-xh Hello! I had the same problem since there is no inference or test script. I made this one[https://github.com/sergiocanar/SegFormer3D_baseline/blob/main/experiments/brats2018/experiment5_final_sweep/inference.py](inference.py) that is on my forked repository of SegFormer3D. However I'm not sure if its  the same that was used in the original implementation since I did not obtain similar metrics to the ones reported on the paper. Let me know if you guys figure it out or have any comments on my inference script
